### DISCUSSION TITLE
Improving best snapshot for bird feeder

### DISCUSSION THREAD

**Discussion by hursey013** [blakeblackshear/frigate#1050] [Ask A Question] [answered]
Hello! I have Frigate working with a bird feeder of mine, capturing snapshots of birds. This has worked great so far (https://twitter.com/casadelcardinal), however sometimes it seems like the best image is not selected. I have watched the feeder when a bird is visiting and can see many much clearer opportunities for snapshots, but ultimately a snapshot will be chosen of the bird turned around and leaving or otherwise obscured.

I understand that in the mind of object detection a bird is a bird whether it's looking at the camera or not, so I suspect even though I may think it's a better shot, from a top score point of view the other less desirable snapshots may have the same or better score. With that said, I'm wondering if there is anything I can tweak to possibly improve the snapshots I'm getting? I have experimented with `best_image_timeout` and `max_disappeared` thinking that perhaps the image is replaced after the default 60 seconds with a snapshot of a lower score, so I have extended that to `300`. Any other suggestions?

**Comment by blakeblackshear** [comment, marked as answer]
Increasing both of those values should ensure you get the highest scoring image. Don't increase max disappeared too much or you will miss birds if two arrive one after the other. Also, make sure you trigger off the end event.
